src/Nelmio/Alice/Fixtures/Builder/Methods/ListName.php

```diff
@@ -22,7 +22,7 @@ class ListName implements MethodInterface
      */
     public function canBuild($name)
     {
-        return preg_match('#\{([^,]+(\s*,\s*[^,]+)*)\}#', $name, $this->matches);
+        return 1 === preg_match('#\{([^,]+(\s*,\s*[^,]+)*)\}#', $name, $this->matches);
```

Comment:
I would rather see these kinds of things in a separate PR - it's not that they aren't obviously useful, but it's a lot of chatter that makes it much more difficult to review and merge quickly.